api/client/set.go

```diff
@@ -0,0 +1,67 @@
+package client
+
+import (
+	"fmt"
+
+	flag "github.com/docker/docker/pkg/mflag"
+	"github.com/docker/docker/pkg/units"
+	"github.com/docker/docker/runconfig"
+)
+
+func (cli *DockerCli) CmdSet(args ...string) error {
+	cmd := cli.Subcmd("set", "CONTAINER [CONTAINER...]", "Setup/Update resource configs for a container", true)
```

Comment:
I'll redo this PR, and expend this feature to set a container wether it's running or stopped.